modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/macros/PRMMDFRequest.macro

```diff
@@ -479,6 +479,22 @@ definition {
 			value1 = "\RETURN");
 	}
 
+	@summary = "Verify if the Error Message show"
+	macro verifyErrorMessege {
+		if (isSet(AssertElementPresent)) {
+			AssertElementNotPresent(
+				key_errorMessage = ${errorMessage},
+				locator1 = "PRMMDFRequest#ERROR_MESSAGE");
+		}
```

Comment:
Hi @pedroamarall, as your macro is doing just one action, the `isSet` isn't necessary.